sift/packages/disktype.sls

```diff
@@ -0,0 +1,2 @@
+disktype:
```

Comment:
Adjust per first comment on review. Needs to be `sift-package-disktype` with modifications to `pkg.installed` instruction to reference actual name.